lib/common/common/src/rate_limiting.rs

```diff
@@ -1,42 +1,13 @@
-use std::time::{Duration, Instant};
-
-/// A rate of requests per time period.
-#[derive(Debug)]
-pub struct Rate {
```

Comment:
removed the infrastructure to handle arbitrary duration as we explicitly specialized on a per minute function